app/src/main/java/com/amaze/filemanager/utils/Utils.java

```diff
@@ -399,4 +401,13 @@ public static Intent buildEmailIntent(String text) {
     emailIntent.setType("message/rfc822");
     return emailIntent;
   }
+
+  /**
+   * Marks a text input field as mandatory (appends * at end)
+   *
+   * @param field given text input field
+   */
+  public static void makeRequired(TextInputLayout field) {
```

Comment:
Please use extension function in kotlin in a new file called "TextInputLayoutExtensions" so that it is easier to find and use.

Reply:
As I said called "TextInputLayoutExtensions".

Reply:
Why? Do we plan to have more extensions for TextInputLayout? Just extensions for UI components should be enough. Shouldn't segregate more than is required.

Reply:
Extensions are quite common in Kotlin, and probably more will be needed. But I do see the point that right now it is the only one.